src/components/tables/TableConnected.tsx

```diff
@@ -79,14 +111,21 @@ const mapDispatchToProps = (
   onUnmount: () => {
     dispatch(removeTable(ownProps.id));
   },
-  onModifyData: (shouldResetPage: boolean, tableCompositeState: ITableCompositeState, previousTableCompositeState: ITableCompositeState) => {
+  onWillUpdate: (actions: IActionOptions[]) => {
+    if (actions.length) {
+      dispatch(addActionsToActionBar(`${ownProps.id}action-bar`, actions));
+    }
+  },
+  onModifyData: (shouldResetPage: boolean, tableCompositeState: ITableCompositeState,
+                 previousTableCompositeState: ITableCompositeState) => {
     if (ownProps.manual) {
       dispatch(ownProps.manual(ownProps, shouldResetPage, tableCompositeState, previousTableCompositeState));
     } else {
       dispatch(defaultTableStateModifierThunk(ownProps, shouldResetPage, tableCompositeState));
     }
   },
-  onRowClick: (actions: IActionOptions[] = []) => {
+  onRowClick: (actions: IActionOptions[] = [], numberOfSelectedIds: number) => {
+    actions = ownProps.rowsMultiSelect && numberOfSelectedIds >= 2 ? _.filter(actions, (action: IActionOptions) => !!action.grouped) : actions;
```

Comment:
https://github.com/coveo/tsjs#functions--reassign-params i know this won't cause any side effect here, but let's try to stick with what we agreed on